.github/workflows/osrm-backend.yml

```diff
@@ -47,6 +47,16 @@ jobs:
         ./scripts/format.sh && ./scripts/error_on_dirty.sh
         node ./scripts/validate_changelog.js
         npm run docs && ./scripts/error_on_dirty.sh
+  
+  docker-image:
```

Comment:
Checks that we won’t have problem like this again

Reply:
Let's add `continue-on-error: false` to this job in that case.

Reply:
Done.